### ISSUE TITLE
Understanding and documenting a few general behaviors

### ISSUE THREAD

**Issue by fmetze** [srvk/DiViMe#17]
What is the expected way that all of the tools do:

- [x] provide log-files (store in VM? outside VM? compress and or delete? only on fail?)
- [x] treat temporary files (keep them around? delete them?)
- [x] treat really long utterances (like in the separate issue)
- [ ] process multiple files in parallel, if possible (like using sbatch?)
- [x] can be configured in certain ways (like cut-offs, parameters, ...)

**Comment by alecristia** [comment]
I have addressed these (except for the behavior for multiple files) in "instructions for contributors" section of the documentation.
